week2/assign1/script.js

```diff
@@ -0,0 +1,47 @@
+/* 스크롤에 따른 topBtn 투명도 */
+
+const windowHeight = window.innerHeight;
+const topBtn = document.getElementById("top-btn");
+
+document.addEventListener("scroll", () => {
+  topBtn.style.opacity = window.scrollY / windowHeight / 2;
+});
+
+/* 글자수에 따른 더보기 버튼 활성화 및 작동 */
+
+const contents = document.querySelectorAll(".content");
+contents.forEach((content) => {
+  const contentDescription = content.children[1];
+  if (contentDescription.innerText.length >= 85) {
```

Comment:
PR에 적어준 부분인 것 같아서,
```suggestion
  if (contentDescription.clientHeight < contentDescription.scrollHeight) {
    }
```
저는 이런식으로 사용해서 구현했어용 참고 !~!